app/controllers/secrets_controller.rb

```diff
@@ -68,4 +68,10 @@ def batch
 
     render json: result
   end
+
+  def expire
+    authorize :update
+    Secret.update_expiration(@resource.id, nil)
+    head :created
```

Comment:
Without documented conventions it's hard to have any ground to stand on but...

I don't know that I agree with `:created` as the right response here. `204 No Content` is more appropriate, in my opinion. :)

Reply:
I think if we switch to supplying an expiration, then `created` will be the right response (since you're creating an expiration for the secret).

Reply:
@micahlee I disagree here.  An expiration is being conceptualized as an http resource.  You "create" a new one to achive expiration.  So this follow REST semantics.

Reply:
Ok, now that I understand the intention better, I'm fine with the `created` response.

Reply:
The V4 docs on expiration provide a mechanism for setting the variable to expire in the future: https://conjur.docs.apiary.io/#reference/variable/set-expiration.  Should we add similar support, thereby allowing `create` to be a more logical action?

Reply:
This could be a future update as well if it falls out of scope of this feature.